code/datums/design/designs/misc/_misc.dm

```diff
@@ -45,8 +45,13 @@
 	design_unlock = DESIGN_UNLOCK_INTRINSIC
 	build_path = /obj/item/light/bulb/fairy
 
-/datum/prototype/design/misc/glass_ashtray
-	id = "MiscGlassAshtray"
+/datum/prototype/design/misc/ashtray
```

Comment:
don't have to change it on this PR but going forwards we should probably do `ashtray` or `item-ashtray` or something, this naming scheme is from me and i realized that it sucks (this is entirely on me)